src/components/media/Media.scss

```diff
@@ -0,0 +1,30 @@
+@import '../../styles/variables';
+
+// the outermost element
+.bdl-Media {
+    align-items: flex-start;
+    display: flex;
+}
+
+// the media content, ie, an avatar
+.bdl-Media-figure {
+    align-self: flex-start;
+    flex: 0 0 auto;
+    justify-self: flex-start;
+    margin: 0;
+    margin-right: 10px;
```

Comment:
I thought we had a SCSS lint rule against using shorthand and long-hand properties in the same style. It's usually discouraged. We should also consider moving the margin to the `-body`, so everything is pushing itself left rather than pushing its neighbor right.